File: Module 3/3A Javascript/JavaScript Coding Challenge/Task7.js
// Sort an array of numbers in descending order

function sortArrayDescending(arr) {
    return arr.sort((a, b) => b - a);
}

// Sample usage
let numberArray = [3, 1, 4, 1, 5, 9, 2, 6, 5, 3, 5];
let sortedArray = sortArrayDescending(numberArray);
console.log('Original Array:', numberArray);
console.log('Sorted Array:', sortedArray);

File: Module 4/4C_Everyday_Go_To_Python/Python_Coding_Challenge.py/Task7.py
"""
Sort an array of numbers in descending order.
"""
def sort_descending(numbers):
    return sorted(numbers, reverse=True)

# Example usage
numbers_array = [5, 1, 8, 3, 7]
sorted_array = sort_descending(numbers_array)

print("Original Array:", numbers_array)
print("Sorted Array (Descending):", sorted_array)
